src/packages/migrate/src/utils/printDatasource.ts

```diff
@@ -5,15 +5,23 @@ export async function printDatasource(schemaPath: string): Promise<void> {
   const dbInfo = await getDbInfo(schemaPath)
 
   if (dbInfo.dbType) {
-    console.info(
-      chalk.dim(
-        `Datasource "${dbInfo.name}": ${dbInfo.dbType} ${dbInfo.schemaWord} "${
-          dbInfo.dbName
-        }"${dbInfo.schema ? `, schema "${dbInfo.schema}"` : ''} at "${
-          dbInfo.dbLocation
-        }"`,
-      ),
-    )
+    if (dbInfo.dbType === 'PostgreSQL' && dbInfo.dbName === undefined) {
+      dbInfo.dbName = 'postgres'
```

Comment:
I think this should default to the username from the URL instead, to match postgres' defaults.

Reply:
Had a discussion with @Jolg42 , did the investigation, and it turns out quaint (our drivers library) does default to "postgres", while psql defaults to the username. So this is correct.

Reply:
Actually Tom checked and in quaint the default is always `postgres` (it doesn't depend on the username, while psql does)